client/my-sites/plugins/plugin-management-v2/utils/get-sites-that-can-install-plugins.ts

```diff
@@ -0,0 +1,19 @@
+import { WPCOM_FEATURES_INSTALL_PLUGINS } from '@automattic/calypso-products';
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+import { IAppState } from 'calypso/state/types';
+import type { SiteDetails } from '@automattic/data-stores';
+
+/**
+ * Filters an array of sites to return only the sites that have the capability to install plugins.
+ * @param state - The application state.
+ * @param sites - An array of SiteDetails objects representing the sites to filter.
+ * @returns An array of SiteDetails objects that can install plugins.
+ */
+export function getSitesThatCanInstallPlugins(
+	state: IAppState,
+	sites: Array< SiteDetails | null | undefined >
+) {
+	return sites.filter( ( site ) => {
+		return siteHasFeature( state, site?.ID, WPCOM_FEATURES_INSTALL_PLUGINS );
+	} );
+}
```

Comment:
I agree we should show those sites to promote upsells. I filtered them to cope with the following requirement in the [parent task](https://github.com/Automattic/dotcom-forge/issues/8082#issue-2390130729)
>  Only show sites where I have permissions to install a plugin

Reply:
> . I filtered them to cope with the following requirement in the https://github.com/Automattic/dotcom-forge/issues/8082#issue-2390130729

Oh I see, I mentioned this because we've made the opposite action not long ago, and we decided to show free sites there to promote upsells.

> But I think this deserves further discussion so I am going to remove it for this change, and continue the discussion in a follow-up task.

I agree with this, thanks